### PR TITLE
doc: update readme

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [ ] `npm test` passes
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Affected core subsystem(s)

document, readme

### CODE REVIEW

**Comment by atian25** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -24,23 +24,48 @@ A web framework's framework for Node.js.
...
 ## Getting Started
 
+Follow the step
+
+```bash
+$ npm install egg-init -g
+$ egg-init --type simple showcase && cd showcase
+$ npm install
+$ npm run dev
+$ open http://localhost:7001
```
Comment (fengmk2) [inline README.md]: http://localhost:7001/news 吧

**Comment by fengmk2** [comment]
+1

File: README.md (hunk 1 above)
Comment (fengmk2) [inline README.md]: @atian25 / 直接路由到 news 好了。

Comment (atian25) [inline README.md]: 好

Comment (atian25) [inline README.md]: https://github.com/eggjs/egg-boilerplate-simple/pull/9

**Pull request merged**